src/pudl_archiver/archivers/doe/doeiraec.py

```diff
@@ -0,0 +1,35 @@
+"""Download DoE IRA Community Energy data."""
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+BASE_URL = "https://edx.netl.doe.gov/dataset/ira-energy-community-data-layers"
+
+
+class DOEIRAECArchiver(AbstractDatasetArchiver):
+    """DOE IRA EC archiver."""
```

Comment:
Probably good to spell out Energy Community as in the other spots

```suggestion
    """DOE IRA Energy Community archiver."""
```